Card.py

```diff
@@ -0,0 +1,83 @@
+import trello as trello
+
+toDoListId = trello.get_trello_list_id("To Do")
+doingListId = trello.get_trello_list_id("Doing")
+doneListId = trello.get_trello_list_id("Done")
+
+
+class Card:
+    def __init__(self, id, name, desc, due):
+        self.id = id
+        self.name = name
+        self.desc = desc
+        self.due = due
+
+
+def get_cards():
+    todo = trello.get_trello("lists/" + toDoListId + "/cards?")
+    doing = trello.get_trello("lists/" + doingListId + "/cards?")
+    done = trello.get_trello("lists/" + doneListId + "/cards?")
+
+    return convertToArrayOfCards(todo), convertToArrayOfCards(doing), convertToArrayOfCards(done)
+
+
+def convertToArrayOfCards(items):
+    returnList = []
+    for item in items:
+        newItem = convertToCard(item)
+        returnList.append(newItem)
+    return returnList
```

Comment:
You could write this function as just `return [ convertToCard(item) for item in items ]`.  But up to you what looks nicest.